notebooks/patch_to_point/patch_to_point.ipynb

```diff
@@ -0,0 +1,689 @@
+    "We can use `query_ground_truth` to create a GeoParquet file to a specific output folder with all points in the RDM that are within a specified Polygon. The Polygon will be constructed from the job_df geometry."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create_datacube"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, create the `create_datacube` function to be given to the GFMapJobManager is created. For this we cannot use worldcereal_classification's `worldcereal_preprocessed_inputs` in it's current form. Maybe best to discuss how we unify as much as possible the preprocessing of the different scripts.\n",
+    "\n",
+    "METEO is loaded from a pre-composited STAC collection --> we have to think about how we keep this up-to-date (maybe include a trigger in the automatic RDM API interaction?)\n",
+    "\n",
+    "In the final script we will include customizable job options, but we first need to figure out the optimal options "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openeo\n",
+    "import pandas as pd\n",
+    "\n",
+    "from openeo_gfmap import BackendContext, FetchType, TemporalContext\n",
+    "from openeo_gfmap.spatial import BoundingBoxExtent\n",
+    "from openeo_gfmap.preprocessing.compositing import median_compositing\n",
+    "from worldcereal.openeo.preprocessing import raw_datacube_DEM\n",
+    "\n",
+    "def create_datacube(row: pd.Series, connection: openeo.Connection, provider=None, connection_provider=None):\n",
+    "    bbox = row.geometry[0]\n",
+    "    spatial_extent = BoundingBoxExtent(bbox[0], bbox[1], bbox[2], bbox[3])\n",
+    "    temporal_context = TemporalContext(row.start_date, row.end_date)  # In the end we'll probably don't need this. Leaving this in, just in case for now.\n",
+    "\n",
+    "    backend = Backend.TERRASCOPE\n",
+    "    backend_context = BackendContext(backend)\n",
+    "\n",
+    "    fetch_type = FetchType.TILE\n",
+    "\n",
+    "    # The S1 and S2 cubes are loaded from their respective STAC collections\n",
+    "    s1_cube = connection.load_stac(row.s1_collection_path, temporal_extent=[row.start_date, row.end_date])\n",
+    "    s1_cube = median_compositing(s1_cube, period=\"month\")\n",
+    "    \n",
+    "    s2_bands = [\n",
+    "            \"S2-L2A-B01\",\n",
+    "            \"S2-L2A-B02\",\n",
+    "            \"S2-L2A-B03\",\n",
+    "            \"S2-L2A-B04\",\n",
+    "            \"S2-L2A-B05\",\n",
+    "            \"S2-L2A-B06\",\n",
+    "            \"S2-L2A-B07\",\n",
+    "            \"S2-L2A-B08\",\n",
+    "            \"S2-L2A-B8A\",\n",
+    "            \"S2-L2A-B11\",\n",
+    "            \"S2-L2A-B12\"\n",
+    "        ]  # Not extracting SCL and distance to cloud, can easily be added if needed\n",
```

Comment:
so how are we applying the mask_scl_dilation cloud mask then now?

Reply:
Is the mask not already applied in the patch extractions? https://github.com/WorldCereal/worldcereal-classification/blob/main/src/worldcereal/openeo/preprocessing.py#L369

Reply:
@GriffinBabe is that so? I though the point of extracting the scl dilation mask and add it to extractions was that we could still switch to e.g. the DL-based mask once it would become mature. Does it now turn out we're masking already with a fixed mask?!

Reply:
Ah sorry, my bad! For the patch extractions the masking is turned off: https://github.com/WorldCereal/worldcereal-classification/blob/main/scripts/extractions/patch_extractions/extract_optical.py#L127

I'll add the scl mask as an extra band then.

Reply:
Just in time to save me from some bad sleep 🤣